gcc/doc/extend.texi

```diff
@@ -21614,6 +21614,82 @@ vector intrinsic specification, which is available at the following link:
 @uref{https://github.com/riscv-non-isa/rvv-intrinsic-doc/tree/v0.11.x}.
 All of these functions are declared in the include file @file{riscv_vector.h}.
 
+These built-in functions are available for the CORE-V MAC machine
+architecture. For more information on CORE-V built-ins, please see
+@uref{https://github.com/openhwgroup/core-v-sw/blob/master/specifications/corev-builtin-spec.md#listing-of-hardware-loop-builtins-xcvhwlp}
```

Comment:
This should go in it's own subsection e.g. `CORE-V Vendor Intrinsics`.

URL mentions `hardware-loop-builtins`?

Reply:
Changed